tests/foreman/destructive/test_infoblox.py

```diff
@@ -232,27 +237,26 @@ def test_infoblox_end_to_end(
     # check hostname and ip is present in A record
     assert host.name in result.text
     assert host.ip in result.text
+    # delete host
+    module_target_sat.api.Subnet(id=subnet.id, domain=[]).update()
+    module_target_sat.api.Host(id=host.id).delete()
+    module_target_sat.api.Subnet(id=subnet.id).delete()
+    module_target_sat.api.Domain(id=domain.id).delete()
+    with pytest.raises(HTTPError):
+        host.read()
```

Comment:
Shouldn't this be part of request.addfinalizer?

Reply:
Not necessarily..we are also checking the deletion of the host. Also after disabling plugin the dhcp and dns feature is disabled so the host cannot be deleted.